usr/share/bash-completion/completions/kapacitor

```diff
@@ -165,6 +164,38 @@ _kapacitor()
     return 0
 }
 
+_kapacitor_files_add_postfix()
+{
+	#Add a slash (directories) or space (files except directories) after name
+	local c i=0
+	for c in "${COMPREPLY[@]}"; do
+		if [ -d "${c}" ]; then
+			COMPREPLY[i++]="${c}/";
+		else
+			COMPREPLY[i++]="${c} ";
+		fi
+	done
+        compopt -o nospace
```

Comment:
The original script didn't support directory auto completion. So one had to adapt the compgen part to get filenames and directories. Per default, bash adds a space, so tabbing though a directory tree wouldn't work. So I disabled this via 'nospace' and manually add spaces and slashes.